[DO-NOT-MERGE] [ION-8334] DLB URL Encoding (#385)
mulesoft/docs-runtime-manager: modules/ROOT/pages/lb-architecture.adoc

== DLB URL Encoding

> When DLB receives a request, it first decodes the URI based on standard internet guideline "RFC-3986" to evaluate the URL mapping rules. During this process, it decodes some set of characters, such as alphabetical (%41–%5A and %61–%7A), digital (%30–%39), hyphen (%2D), period (%2E), underscore (%5F), tilde (%7E), slash (%2F), space (%20), octothorpe (%23) etc.

Reviewer: ```suggestion
When DLB receives a request, it first decodes the URI based on standard internet guideline "RFC-3986" to evaluate the URL mapping rules. During this process, it decodes some characters, such as alphabetical (%41–%5A and %61–%7A), digital (%30–%39), hyphen (%2D), period (%2E), underscore (%5F), tilde (%7E), slash (%2F), space (%20), and octothorpe (%23).
```

> * If you select *Keep URL encoding*, DLB re-encodes these decoded URI and pass the request to the Cloudhub worker.

Reviewer: ```suggestion
* If you select *Keep URL encoding*, DLB re-encodes the decoded URI and passes the request to the CloudHub worker.
```

> * If you deselect *Keep URL encoding*, the DLB passes the decoded URI as is to the Cloudhub worker.

Reviewer: ```suggestion
* If you deselect *Keep URL encoding*, the DLB passes the decoded URI as is to the CloudHub worker.
```
